src/ZipkinExporter.php

```diff
@@ -161,7 +184,8 @@ public function convertSpans(array $spans, $headers = null)
                 : null;
             $traceId = str_pad($span->traceId(), 32, '0', STR_PAD_LEFT);
 
-            $attributes = $span->attributes();
+            $attributes = array_merge($this->defaultAttributes, $span->attributes());
```

Comment:
Same as my other comment about ordering, should we let our default attributes be overwritten?